core/src/main/java/org/sonarsource/sonarlint/core/ConnectedSonarLintEngineImpl.java

```diff
@@ -91,13 +90,13 @@ public void setVerbose(boolean verbose) {
     }
   }
 
-  public GlobalContainer getGlobalContainer() {
+  public StorageGlobalContainer getGlobalContainer() {
     return globalContainer;
   }
 
   public void start() {
     rwl.writeLock().lock();
-    this.globalContainer = GlobalContainer.create(globalConfig);
+    this.globalContainer = StorageGlobalContainer.create(globalConfig);
     try {
       globalContainer.startComponents();
       if (globalContainer instanceof StorageGlobalContainer) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless "instanceof" operator; it will always return "true". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1850)